README.md

```diff
@@ -78,7 +78,7 @@ $ git clone https://github.com/vmware/tern.git
 $ source bin/activate
 $ cd tern
 $ pip install .
-$ tern -l report -f output.txt -i debian:buster
+$ python -m tern -l report -f output.txt -i debian:buster
```

Comment:
What is the difference?

Reply:
On my Ubuntu system, I'm getting
```
(venv) $ tern -l report -f output.txt -i debian:buster
No command 'tern' found, did you mean:
 Command 'etrn' from package 'sendmail-base' (universe)
 Command 'trn' from package 'trn' (multiverse)
 Command 'trn' from package 'trn4' (multiverse)
 Command 'terd' from package 'tcm' (universe)
tern: command not found
```

Reply:
Interesting. The `.travis.yml` uses xenial for testing and the tests are passing.

Reply:
These set of commands worked for me - Fedora 29 VM
```
$ python3 -m venv testern
$ cd testern/
$ source bin/activate
$ git clone git@github.com:vmware/tern.git
$ cd tern
$ pip install . <-- where setup.py is
$ tern -l report -i centos:7
```